contracts/BEP20Interface.sol

```diff
@@ -1,4 +1,4 @@
-pragma solidity ^0.5.16;
+pragma solidity 0.5.17;
```

Comment:
you consider this update will be compatible with current deployed contract?

Reply:
current contracts are compiled and deployed with 0.5.17, that's why no problem for these contracts.
but need to confirm proxy delegate calls and storage usages for these update.